python/paddle/nn/functional/loss.py

```diff
@@ -1471,6 +1473,113 @@ def nll_loss(
         return out
 
 
+def poisson_nll_loss(
+    input,
+    label,
```

Comment:
It is recommended to change [rfc](https://github.com/PaddlePaddle/community/blob/master/rfcs/APIs/20230301_api_design_for_poissonnllloss.md) parameter `target` to `label`, so keep consistent here.

Reply:
We update our API parameter in our rfc and create a new [pr](https://github.com/PaddlePaddle/community/pull/484).